requirements/dev.txt

```diff
@@ -0,0 +1,235 @@
+#
+# This file is autogenerated by pip-compile with python 3.6
+# To update, run:
+#
+#    pip-compile --output-file=requirements/dev.txt requirements/dev.in
+#
+-e git+https://github.com/kytos-ng/kytos.git#egg=kytos
+    # via -r requirements/dev.in
+-e .
+    # via -r requirements/dev.in
+-e git+https://github.com/kytos-ng/of_core.git@fix/action_base_from_dict#egg=kytos_of_core
```

Comment:
This is a temporary commit, I'll revert this commit https://github.com/kytos-ng/noviflow/pull/3/commits/6bf478c2ae9e5cc8bc644696e1f1e9daabbfaf1b pointing back to `of_core` master branch before it gets approved, so it depends on getting merged https://github.com/kytos-ng/of_core/pull/36

Reply:
@ajoaoff I've just reverted this commit on https://github.com/kytos-ng/noviflow/pull/3/commits/feecc490744dd5d9ecfd8b4fe79f0cf4fe1de77c, and I've also applied some `pylint` fixes https://github.com/kytos-ng/noviflow/pull/3/commits/7049c1ae1f3f74466ab57115cc2d6a0df9ce7d31, could you do a second pass review? thanks